ui/src/actions/apps.js

```diff
@@ -36,3 +36,23 @@ export const deleteApp = name => fetchAction(
   [APP_DELETE_REQUEST, APP_DELETE_SUCCESS, APP_DELETE_FAILURE],
   async () => DataService.deleteApp(name),
 )();
+
+export const registerPlatform = (platform) => ({
+  type: 'platform/REGISTER',
+  platform: { name: platform.name, selected: platform.selected } 
```

Comment:
Is `selected` needed here? It is not used in reducer [1].

[1] https://github.com/aerogear/mobile-developer-console/pull/96/commits/11e2eb36803253cf2ddebd3f9e0a51bafb699525#diff-a7113868962bf92abdf2145b05c18bbfR143